
A GCC cross compiler does not yield a libgcc_eh.so. The GCC and GlibC team
should smoke their good stuff together (and the GlibC freaks be fet with a
book about release management ...).

  - Rene Rebe <rene@rocklinux.org>

--- glibc-2.3.4-2004-05-01/Makeconfig	2004-04-18 00:38:24.000000000 +0200
+++ glibc-2.3.4-2004-05-01-fixed/Makeconfig	2004-05-03 23:25:31.000000000 +0200
@@ -518,16 +518,16 @@
 
 ifndef gnulib
 ifneq ($(have-as-needed),yes)
- libgcc_eh := -lgcc_eh
+ libgcc_eh := 
 else
  libgcc_eh := -Wl,--as-needed -lgcc_s -Wl,--no-as-needed
 endif
 ifneq ($(have-cc-with-libunwind),yes)
  gnulib := -lgcc $(libgcc_eh)
- static-gnulib := -lgcc -lgcc_eh
+ static-gnulib := -lgcc $(libgcc_eh)
 else
  gnulib := -lgcc $(libgcc_eh) -lunwind
- static-gnulib := -lgcc -lgcc_eh -lunwind
+ static-gnulib := -lgcc $(libgcc_eh) -lunwind
 endif
 endif
 ifeq ($(elf),yes)
